src/components/background/Background.tsx

```diff
@@ -0,0 +1,138 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+import Bubble from './Bubble';
+import { flags } from '../../assets/flags';
+
+const BgStyled = styled.div`
+  /* z-index: -1; */
+  /* height: 100vh; */
+  /* width: 100vw; */
+  /* background-attachment: fixed;
+  background-image: radial-gradient(
```

Comment:
wszystkie komentarze do wywalenia

Reply:
Przyjdzie na nie czas, to faza testów.

Reply:
często sie później o takiej rzeczach zapomina. ale dobra :D